index.bs

```diff
@@ -4930,7 +4930,7 @@ in order to allow users to [=authentication|authenticate=] using their registere
      of the {{CredentialsContainer/get()}} method:
 
      - Set the {{PublicKeyCredentialDescriptor/type}} members to {{PublicKeyCredentialType/public-key}}.
-     - Set the {{PublicKeyCredentialDescriptor/id}} members to the respective U2F key handles of the desired credentials.
+     - Set the {{PublicKeyCredentialDescriptor/id}} members to the respective U2F key handles of the desired credentials. Note that U2F key handles are commonly base64url encoded but must be decoded to their binary form when used in {{PublicKeyCredentialDescriptor/id}}.
```

Comment:
suggest:
s/ base64url encoded / [=base64url encoding|base64url encoded=] /

note: using just [=base64url encoded=] may do the right thing, would have to try it and see.

Reply:
> 
> 
> suggest:
> s/ base64url encoded / [=base64url encoding|base64url encoded=] /
> 
> note: using just [=base64url encoded=] may do the right thing, would have to try it and see.

No it wouldn't. "Base64url Encoding" is a specifically-defined term in WebAuthn, and base64url encoding uses padding at the end unless stated otherwise by the specification that uses it (see RFC 4648 section 3.2), for example, in WebAuthn's "Base64url Encoding" definition.

Reply:
Have used `[=base64url encoding=]`. It links [here](https://w3c.github.io/webauthn/#base64url-encoding) in the WebAuthn spec, which defines a base64 encoding with padding removed, which is correct for U2F.